src/main/java/io/kestra/plugin/mongodb/Aggregate.java

```diff
@@ -0,0 +1,254 @@
+                    connection:
+                      uri: "mongodb://root:example@localhost:27017/?authSource=admin"
+                    database: "my_database"
+                    collection: "sales"
+                    pipeline:
+                      - $match:
+                          status: "active"
+                      - $group:
+                          _id: "$category"
+                          total: 
+                            $sum: "$amount"
+                          count:
+                            $sum: 1
+                      - $sort:
+                          total: -1
+                """
+        ),
+        @Example(
+            full = true,
+            title = "Complex aggregation with lookup and data transformation",
+            code = """
+                id: mongodb_complex_aggregate
+                namespace: company.team
+                
+                tasks:
+                  - id: aggregate_with_lookup
+                    type: io.kestra.plugin.mongodb.Aggregate
+                    connection:
+                      uri: "mongodb://root:example@localhost:27017/?authSource=admin"
+                    database: "my_database"
+                    collection: "users"
+                    pipeline:
+                      - $lookup:
+                          from: "orders"
+                          localField: "_id"
+                          foreignField: "userId"
+                          as: "userOrders"
+                      - $addFields:
+                          totalOrders:
+                            $size: "$userOrders"
+                          totalSpent:
+                            $sum: "$userOrders.amount"
+                      - $project:
+                          name: 1
+                          email: 1
+                          totalOrders: 1
+                          totalSpent: 1
+                      - $match:
+                          totalOrders:
+                            $gt: 0
+                    allowDiskUse: true
+                    maxTimeMs: 30000
+                """
+        )
+    }
+)
+public class Aggregate extends AbstractTask implements RunnableTask<Aggregate.Output> {
+    @Schema(
+        title = "MongoDB aggregation pipeline.",
+        description = "List of pipeline stages as a BSON array or list of maps."
+    )
+    @PluginProperty(dynamic = true)
+    private List<Object> pipeline;
+
+    @Schema(
+        title = "Whether to allow disk usage for stages.",
+        description = "Enables writing to temporary files when a pipeline stage exceeds the 100 megabyte limit."
+    )
+    @Builder.Default
+    private Property<Boolean> allowDiskUse = Property.ofValue(false);
+
+    @Schema(
+        title = "Maximum execution time in milliseconds.",
+        description = "Sets the maximum execution time on the server for this operation."
+    )
+    @Builder.Default
+    private Property<Integer> maxTimeMs = Property.ofValue(0);
```

Comment:
Maybe increase the default value since if the user forget it the task will end as soon as it's run no?